Terminal.Gui/ConsoleDrivers/ConsoleDriver.cs

```diff
@@ -608,6 +613,66 @@ public void OnMouseEvent (MouseEventArgs a)
     /// <param name="ctrl">If <see langword="true"/> simulates the Ctrl key being pressed.</param>
     public abstract void SendKeys (char keyChar, ConsoleKey key, bool shift, bool alt, bool ctrl);
 
+    /// <summary>
+    ///     Provide handling for the terminal start reporting mouse events.
+    /// </summary>
+    public abstract void StartReportingMouseMoves ();
+
+    /// <summary>
+    ///     Provide handling for the terminal stop reporting mouse events.
+    /// </summary>
+    public abstract void StopReportingMouseMoves ();
```

Comment:
Its called `StopReportingMouseMoves` but really what we want is for the driver to ensure nothing comes over std in right? Presume even things like resize console or console gains focus will also be stopped for duration of this ANSI request class lifetime?

If so should this just be called `Suspend` or something? And have it full stop everything?

Reply:
I want to ensure that only mouse events are disabled over std-in but I ensure the existent one are processed before the request, because they're the biggest ansi escape sequence in std-in and take a long time to be processed. Any other events like resize or focus are processed after the request return.
Since it's only disable and enable mouse events I think the current name is correct.